kube-config-files/image-cleaner.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: image-cleaner
+  labels:
+    app: image-cleaner
+spec:
+  selector:
+    matchLabels:
+      app: image-cleaner
+  template:
+    metadata:
+      labels:
+        app: image-cleaner
+        visualize: "true"
+    spec:
+      containers:
+      - name: image-cleaner
+        image: coco/docker-gc:add-loop
+        volumeMounts:
+        - name: host-docker-sock
+          mountPath: /var/run/docker.sock
+        resources:
+          limits:
+            memory: 256Mi
+        env:
+          - name: TIME_BETWEEN_RUNS
```

Comment:
This looks good.
I'm putting this comment just for us to know this is possible: As a nice to have feature, we could read this value every time from a file, that could be mounted from a config map, so that we can change the value at runtime without restarting the app. When changing the value in a ConfigMap, Kubernetes makes sure that the file mounted will be updated.